ethereum/contracts/Splitter.sol

```diff
@@ -25,7 +22,7 @@ contract Splitter {
     emit SplitHandle(msg.sender, receiver1, receiver2);
```

Comment:
Should include msg.value so we could, in thoery, recreate the state history of the contract from logs.